Spring_Backend/src/main/java/polsl/tab/skiresort/api/technical_employee/service/TechnicalEmployeeService.java

```diff
@@ -0,0 +1,26 @@
+package polsl.tab.skiresort.api.technical_employee.service;
+
+import org.springframework.stereotype.Service;
+import polsl.tab.skiresort.api.technical_employee.request.SkiLiftRequest;
+import polsl.tab.skiresort.api.technical_employee.response.SkiLiftResponse;
+import polsl.tab.skiresort.repository.SkiLiftRepository;
+import org.springframework.web.server.ResponseStatusException;
+import org.springframework.http.HttpStatus;
+
+@Service
+public class TechnicalEmployeeService {
+
+    private final SkiLiftRepository skiLiftRepository;
+
+    public TechnicalEmployeeService(SkiLiftRepository skiLiftRepository){this.skiLiftRepository = skiLiftRepository;}
+
+    public SkiLiftResponse editSkiLiftIsOpened(Integer skiLiftId, SkiLiftRequest skiLiftRequest)
+    {
+        var skiLift = skiLiftRepository.findById(skiLiftId)
+                .orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND,"Ski lift does not exist"));
+        skiLift.setName(skiLiftRequest.getName());
```

Comment:
There is no need to rename ski lift, technical will only change true/false flag in database